cmd/aspect/run/run.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright © 2021 Aspect Build Systems
+
+Not licensed for re-use
+*/
+
+package run
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"aspect.build/cli/pkg/aspect/run"
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/interceptors"
+	"aspect.build/cli/pkg/ioutils"
+	"aspect.build/cli/pkg/plugin/system"
+	"aspect.build/cli/pkg/plugin/system/bep"
+)
+
+// NewDefaultRunCmd creates a new run cobra command with the default
+// dependencies.
+func NewDefaultRunCmd(pluginSystem system.PluginSystem) *cobra.Command {
+	return NewRunCmd(
+		ioutils.DefaultStreams,
+		pluginSystem,
+		bazel.New(),
+	)
+}
+
+func NewRunCmd(
+	streams ioutils.Streams,
+	pluginSystem system.PluginSystem,
+	bzl bazel.Bazel,
+) *cobra.Command {
+	return &cobra.Command{
+		Use:   "run",
+		Short: "Builds the specified target and runs it with the given arguments.",
+		Long: `'run' accepts any 'build' options, and will inherit any defaults
+provided by .bazelrc.
+
+TODO(f0rmiga): the following comment from 'bazel --help run' may not be what we
```

Comment:
Can this be a comment, seems odd to include a todo in the help output.

Reply:
Fixed in ad32734.